client/scala/src/main/scala/ai/verta/blobs/dataset/Dataset.scala

```diff
@@ -38,4 +42,13 @@ trait Dataset extends Blob {
 
   /** Get the set of all the files' metadata managed by the Dataset blob  */
   def getAllMetadata = contents.values
+
+  /** Check if the other dataset is combinable (i.e no conflicting entries)
+   *  @param other other dataset to combine
+   *  @return whether there is a conflict in the contents of two dataset
+   */
+  protected def conflicts(other: Dataset) = {
```

Comment:
Actually, this is checking for `notConflicts`, right? I read wrong the first time, I think.

Reply:
Fixed.